panel/models/anywidget_component.ts

```diff
@@ -0,0 +1,83 @@
+import type * as p from "@bokehjs/core/properties"
+
+import {ReactComponent, ReactComponentView} from "./react_component"
+
+const panel_adapter = `
+class PanelModel {
+
+  constructor(model) {
+    this.model = model;
+  }
+
+  get(name) {
+    let value
+    if (name in this.model.attributes) {
+      value = this.model[name]
+    } else {
+      value = this.model.data[name]
+    }
+    if (value instanceof ArrayBuffer) {
+      value = new Uint8Array(value)
+    }
+    return value
+  }
+
+  set(name, value) {
+    return this.data[name] = value;
+  }
+
+  save_changes() {
+    // no-op, not a thing in panel
+  }
+
+  on(event, cb) {
```

Comment:
Don't we need to support more than `change` events. When `AnyWidget` supports it the `AnyWidget` `esm` will contain it. That is what I've seen in practice.

Reply:
Bokeh 3.5 will add support for server side events, once there's a dev release (likely tomorrow) we can use we can leverage them here.